bn-expansions/src/base/em-rule-base.ts

```diff
@@ -10,6 +10,7 @@ export interface ExpansionParameters {
     audit?: Catches
     fishTickets?: FishTicketRow[];
     mixedGroupings?: any;
+    speciesCodeLookup?: any;
```

Comment:
is this going to be what's fetched from em-views/wcgopCode-to-pacfinCode-map? Also is  there an example of how this is going to be called?

Reply:
yep, will do.  once I've published bn-expansions I'll add to trips-api and send a pr.

Reply:
yep, will do